api-server/server/boot/donate-paypal.js

```diff
@@ -0,0 +1,134 @@
+/* eslint-disable max-len */
```

Comment:
Lets fix the cause for this, and not the symptom.

Reply:
Also, why do we need additional boot script for PayPal?

Reply:
just to reduce the size of the donate I moved the paypal related logic to another file which could probably be placed in a helper directory.

Please let me know of your preferred way to proceed.

Reply:
Loopback hooks the boot scripts in the boot folder (presumably in alphabetical order). The size of the file does not matter on the server side of things unless it is meant to be shared across other places.

Reply:
I will include all donation related logic in the same file then.